src/main/scala/com/ambiata/disorder/PositiveLongSmall.scala

```diff
@@ -0,0 +1,16 @@
+package com.ambiata.disorder
+
+import org.scalacheck._
+import scalaz._, Scalaz._
+
+case class PositiveLongSmall(value: Long)
```

Comment:
I'm not sure I see much point in this - it's never going to be larger than `Int`, in which case is `PositiveIntSmall` enough?